### PR TITLE
Add Jest testing capabilities

### PR DESCRIPTION
**Pull request by ananyeet**
### What is a quick description of the change?

Add testing capabilities using [Jest](https://jestjs.io/en/). 

### Is this fixing an issue?

none

### Are there more details that are relevant?

This PR also includes a couple of sample tests just to have an example of how to write a Jest test. There is also a README update. 

### Check lists (check `x` in `[ ]` of list items)

- [x] Test written/updating
- [x] Tests passing
- [x] Coding style (indentation, etc)

### CODE REVIEW

File: slurp/src/test/sum.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+function sum(a, b) {
```
Comment (beiebi0) [inline slurp/src/test/sum.js:1]: Worth to have some comments here to explain this is a sample for using jest as you mentioned that in the RP description. This looks irrelevant to this project which might create confusion.

File: slurp/src/test/sum.test.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+const sumfunctions = require('./sum.js');
```
Comment (beiebi0) [inline slurp/src/test/sum.test.js:1]: Ditto.

File: README.md [hunk 3]
```diff
@@ -2,3 +2,27 @@
...
+## Testing
+slurp/src/test includes tests for this project using Jest. To use it,
+first install npm, then run the following:
+
+```
+npm install --save-dev test
+npm install jest --global
+
+npm install --save-dev jest babel-jest @babel/preset-env 
+
```
Comment (Hiram-Silvey) [inline README.md:15]:
```suggestion
```
